pmm-app/tests/verifyPMMSettingsPageElements_test.js

```diff
@@ -166,5 +141,50 @@ xScenario(
 
     await pmmSettingsPage.waitForPmmSettingsPageLoaded();
     I.waitForValue(pmmSettingsPage.fields.dataRetentionCount, dataRetention, 30);
-  }
+  },
+);
+
+Scenario(
+  'PMM-T415 - Verify Percona Platform (Sign up) elements on PMM Settings Page',
+  async (I, pmmSettingsPage) => {
+    await pmmSettingsPage.waitForPmmSettingsPageLoaded();
+    I.waitForElement(pmmSettingsPage.fields.perconaPlatformLink, 30);
+    I.click(pmmSettingsPage.fields.perconaPlatformLink);
+    I.waitForElement(pmmSettingsPage.fields.singInToSignUpButton, 30);
+    I.click(pmmSettingsPage.fields.singInToSignUpButton);
+    I.waitForElement(pmmSettingsPage.fields.signUpEmail, 30);
+    I.waitForElement(pmmSettingsPage.fields.signUpPassword, 30);
+    const agreementLabel = await I.grabTextFrom(pmmSettingsPage.fields.signUpAgreementLabel);
+
+    assert.ok(
```

Comment:
It will not work this way. It should look like `assert.ok(agreementLabel === pmmSettingsPage.agreementText, `${agreementLabel}: This is not correct agreement label` )`